openedx/stanford/lms/envs/common.py

```diff
@@ -69,6 +69,7 @@
 INSTALLED_APPS += (
     'branding_stanford',
     'cme_registration',
+    'openedx.stanford.lms.djangoapps.instructor',
```

Comment:
Do we need to install our `instructor_tasks` app too?
Are celery tasks discovered automatically?